include/seastar/core/timer-set.hh

```diff
@@ -66,7 +66,7 @@ private:
         return _time_point.time_since_epoch().count();
     }
 
-    static timestamp_t get_timestamp(Timer& timer) noexcept
+    static timestamp_t get_timestamp(Timer const& timer) noexcept
```

Comment:
`Timer const&` is equivalent to `const Timer&`, and we usually use the latter format. I also personally find it much more "obvious" and readable.

Reply:
fixed